src/core/utils/helpers/general.ts

```diff
@@ -224,4 +226,43 @@ export const materialIsFiction = ({
   fictionNonfiction
 }: Work | Manifestation) => fictionNonfiction?.code === "FICTION";
 
+interface PageSizeDataAttributes {
+  desktop: number;
+  mobile: number;
+}
+
+export const getPageSizeFromDataAttributes = ({
+  desktop,
+  mobile
+}: PageSizeDataAttributes) => {
+  const { pageSize } = getDeviceConf("pageSize", {
+    pageSize: {
+      mobile: {
+        pageSize: mobile
+      },
+      desktop: {
+        pageSize: desktop
+      }
+    }
+  });
+  return Number(pageSize);
+};
+
+export const getPageSize = (
+  pageSizes: Partial<PageSizeDataAttributes>,
+  configName: ConfScope
+) => {
+  const { desktop, mobile } = pageSizes;
+  let pageSize = 0;
+  if (desktop && mobile) {
+    pageSize = getPageSizeFromDataAttributes(
+      pageSizes as PageSizeDataAttributes
```

Comment:
Is `as` necessary here?
If it is, maybe it could be solved in the if clause by checking:
```
if (pageSizes?.desktop && pageSizes?.mobile) {